src/openMVG/features/svgVisualization.cpp

```diff
@@ -15,7 +15,7 @@
 #include <algorithm> 
```

Comment:
It compiles without `#include <algorithm>` but if I remove `#include <openMVG/localization/CCTagLocalizer.hpp>` it cannot. What is the header you're talking about ?